Informal List of Projects

```diff
@@ -0,0 +1,66 @@
+###Code Repository
+A kind of Gitbuh but decentralised. Rewards based on commits.
+
+###Security Cameras 
+Use case: you can install security cameras in a kindergarten but no-one can access the footage unless 51% of parents give the consent to do it. It would solve the privacy vs security issue.
+
+###Plugin as Coinpayments
+But open, as with myetherwallet, you can add custom token just defining name, address and decimals
+
+###Call-Center / Customer-Care on demand
+Uber economy; Decentralised customer care on demand.
+It would be also possible to create services for marketplaces (i.e. themeforest) where support is based on some reward system
+
+###Products Warranty / delivery tracking / dropshipping infrastructure
+bill/warranty/tracking on the blockchain (QRcode)
+You could also make automated order  fulfillment with smart-contracts for dropshipping services
+
+###Turnover
+Shifts system. I.e. a Netflix licens give me the right to use up to 4 devices but we could pay it in 20 people and use it according to a shared calendar.
+
+###Easy Mobile Investments
+Every day we send users a push notification e they can decide whether to invest 10, 20, 50 o 100 $ in just a click.
+It’s in between betting and investing: instead of betting on Lotto or football, users can quick invest on crypto
+
+###Satellites as a service
+Platform where users can buy services based on a satellite infrastructure.
+Various applications (i.e. security, urban planning, architecture etc)
+There’s some company that makes commercial satellites. I have a contact.
+https://vimeo.com/161656838
+
+###Coupon
+Tool that allows users to easily create Smart Contracts that can be transferable, time-based e non-cumulative
+
+###Collecting points card
+Similar to ENJIN COIN but in this case you can also collect points from supermarket, fly-miliage,  petrol station discounts etc.)
+
+###Private Photo Album
+Save and privately share a beautiful, private collection of photos, videos and thoughts of your children that you and your family will always cherish. Same model as 23snaps but on the blockchain.
```

Comment:
Love it, immutability might be a double edged sword in this case.
Think about unlicensed content, revenge porn and pedophilia.

Reply:
Well, they can already do it with any kind of crypted message. In this case it's more about the UX for mums and family members.